src/Uplink/Admin/Fields/Field.php

```diff
@@ -199,24 +221,32 @@ public function get_classes(): string {
 	/**
 	 * Renders the field.
 	 *
+	 * @return void
+	 */
+	public function render( array $args = [] ): void {
+		echo $this->get_render_html();
+	}
+
+	/**
+	 * Returns the field.
+	 *
 	 * @return string
 	 */
-	public function render(): string {
-		Config::get_container()->get( License_Field::class )->enqueue_assets();
+	public function get_render_html(): string {
+		$this->assets->enqueue_assets();
 
 		if ( $this->resource->is_using_oauth() ) {
 			ob_start();
 			UplinkNamespace\render_authorize_button( $this->get_slug() );
 			return ob_get_clean();
```

Comment:
Rare, but this can technically return false.

```suggestion
			return (string) ob_get_clean();
```